lib/remapify.js

```diff
@@ -30,7 +30,7 @@ setLogger = function setLogger (verbose) {
 // eslint has a bug :(
 transformAliases = function transformAliases (b, results) {
   /* eslint-enable no-shadow*/
-  var expandedAliasesCount = _(results).pluck('aliases').map(_.size).reduce(function sum (total, count) {
+  var expandedAliasesCount = _.map(results, 'aliases').map(_.size).reduce(function sum (total, count) {
```

Comment:
hmmm… If we do this, we loose the "chaining" from lodash. What about `_.chain(results).map('aliases)……`

Reply:
I believe the two function calls here *should* be synonymous. [v4](https://lodash.com/docs/4.17.10#map) vs. [v3](https://lodash.com/docs/3.10.1#pluck)

Since `_.map()` is giving us back an array we should be able to continue the method chaining like it was previously.

Reply:
Right, but won't we loose the benefits of sequences? https://lodash.com/docs/4.17.10#chain

Reply:
I just ran the tests both with and without the `_.chain` and for whatever reason adding in `_.chain` is causing the tests to fail.

Reply:
lol, okay, whatever then :)